draft-ietf-rats-corim.md

```diff
@@ -1591,129 +1591,118 @@ The Verifier SHOULD set the `authorized-by` field in Accepted Claims Set entries
-all Endorsements in the group are silently ignored.
+For each Conditional Endorsement Triple the Verifier compares the `stateful-environment-record` field in the triple against the ACS (see {{sec-match-one-se}}).
 
-Each group is processed independently of other groups. If a group fails to match
-the Accepted Claims Set then this does not affect the processing of other groups.
+If the stateful environment matches then the Verifier adds an Endorsement entry to the ACS (see {{sec-add-to-acs}}).
+The endorsement is created from the `environment-map` and the `measurement-values-map` field in the triple, and the authority which signed the tag containing the Conditional Endorsement Triple.
 
-#### Matching a Reference Value against the Accepted Claims Set {#sec-match-one-ref-val}
+#### Processing Multi-Environment Conditional (MEC) Endorsement Triple
 
-This section describes how a Reference Value is matched against Evidence in the Accepted
-Claims Set.
-If any part of the processing indicates that the Reference Value does not match then the remaining steps in this section are skipped for that group.
+For each MEC Endorsement Triple the Verifier compares each of the `stateful-environment-record` fields from the `cond` field in the triple against the ACS (see {{sec-match-one-se}}).
 
-A Reference Value consists of an `environment-map` plus a `measurement-map`. In the
-`reference-triple-record` these are encoded together. In other triples multiple
-Reference Values are represented more compactly by letting one `environment-map`
-apply to multiple `measurement-map`s.
+If every stateful environment matches then the Verifier adds an Endorsement entry to the ACS (see {{sec-add-to-acs}}) for each `endorsed-triple-record` in the `endorsements` field.
+Each endorsement is created from the `endorsed-triple-record` and the authority which signed the tag containing the MEC Endorsement Triple.
 
-The Verifier first looks for entries in the Accepted Claims Set with the same
-`environment-map` as the Reference Value. These are the candidate claims. If there are
-no candidate claims then the Reference Value does not match.
+#### Processing Conditional Endorsement Series Triple
 
-A Verifier SHALL compare two `environment-map`s using a binary comparison of the CBOR
-encoded objects.
+For each Conditional Endorsement Series Triple the Verifier matches the triple in two steps.
+The first step compares the `stateful-environment-record` field in the triple against the ACS (see {{sec-match-one-se}}). If this matches then the Verifier performs the second step.
 
-A Verifier SHOULD convert `environment-map` into a form which meets CBOR Core
-Deterministic Encoding Requirements {{-cbor}} before performing the binary comparison.
+The second step steps through the `conditional-series-record`s in order, comparing each of them against the ACS, using a temporary stateful environment created from the `environment-map` in step one and the `refv` field.
```

Comment:
The whole second step needs to be clarified!

I would attempt to rephrase it, see if you like it then take it

The second step iterates through the `conditional-series-record`s in order, comparing the `reference-value` in the record against the `reference-value` in the ACS